test/signs_ui/signs/messages_test.exs

```diff
@@ -54,6 +54,20 @@ defmodule SignsUi.Signs.MessagesTest do
              SignsUi.Signs.Messages.list_messages(pid)
   end
 
+  test "when we send a stopped stops away message, only sends the n-stops part" do
+    {:ok, pid} = SignsUi.Signs.Messages.start_link()
+
+    SignsUi.Signs.Messages.add_message(pid, %{
+      "MsgType" => "SignContent",
+      "c" => ["e130~n2-\"Alewife stopped\".4-\"Alewife 4 stops\".4-\"Alewife away\".4"],
+      "sta" => "RDTC",
+      "uid" => "616722"
+    })
+
+    assert %{"RDTC-n" => [%{duration: _, text: ""}, %{duration: _, text: "Alewife 4 stops"}]} =
```

Comment:
Are we really only expecting one space between `Alewife` and `4 stops`?

Reply:
no, but i dont think that matters for this test

Reply:
Oh I get it, duh.  Carry on